### DISCUSSION TITLE
Wrap focusable control

### DISCUSSION THREAD

**Discussion by hydroper** [godotengine/godot-proposals#6322] [GUI]
I've a `VBoxContainer` as a set of `{x, y, z}` buttons. If I'm currently focusing `y` and press `ui_down`, I then focus `z`. If I'm currently focusing `z`, somehow it doesn't focus `x`. How can I focus from `z` to `x` without an utility subset of `VBoxContainer`?

**Comment by AThousandShips** [comment]
By assigning focus neighbors, set the down neighbor of Z to be X

**Reply by hydroper** [reply]
Oh, I thought I had to set focus neighbors at the `VBoxContainer` itself. Now I tried what you said and it worked.
